go/arrow/internal/debug/log_on.go

```diff
@@ -14,6 +14,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//go:build debug
```

Comment:
no space after `//`?

Reply:
no, compiler directives like build constraints explicitly have no space after the `//`